src/promptflow/promptflow/batch/_result.py

```diff
@@ -200,8 +199,13 @@ def _get_node_status(line_results: List[LineResult], aggr_result: AggregationRes
 
 
 def _get_node_run_infos(line_results: List[LineResult], aggr_result: AggregationResult):
-    line_node_run_infos = (
-        node_run_info for line_result in line_results for node_run_info in line_result.node_run_infos.values()
-    )
-    aggr_node_run_infos = (node_run_info for node_run_info in aggr_result.node_run_infos.values())
-    return chain(line_node_run_infos, aggr_node_run_infos)
+    node_run_infos = {node_run_info.node: node_run_info for node_run_info in aggr_result.node_run_infos.values()}
+    for line_result in line_results:
+        for node, node_run_info in line_result.node_run_infos.items():
+            # For resume runs, we only want to include the latest run info for aggregation node
```

Comment:
Do you mean that the node_run_info of aggregation node is included in line result?